liquibase-core/src/main/java/liquibase/change/core/CreateTableChange.java

```diff
@@ -1,6 +1,19 @@
 package liquibase.change.core;
 
-import liquibase.change.*;
+import static liquibase.statement.SqlStatement.EMPTY_SQL_STATEMENT;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import liquibase.change.AbstractChange;
```

Comment:
I've wondered this in many of the classes I've reviewed, but does the documentation require expansion of imports down to the exact class name? The `package.name.*` is a more common annotation when most or all of a package is being imported.

Reply:
`liquibase.change.*` should work! I think this may have been my IDE throwing a fit...